pyvrp/Model.py

```diff
@@ -181,6 +181,13 @@ def add_edge(
         if distance < 0 or duration < 0:
             raise ValueError("Cannot have negative edge distance or duration.")
 
+        if max(distance, duration) > MAX_USER_VALUE:
```

Comment:
I moved this to `Model.add_edge`, which seemed like a more logical place for it.